test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1956,6 +1958,28 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
             expect(test.form.el('factor-question').length).toEqual(1);
           });
       });
+      itp('calls transaction.authenticate with username, options and stateToken', function () {
```

Comment:
I think we should rename this test to something that explicitly says we have stateToken from the config. Maybe `calls transaction.authenticate with stateToken when stateToken is in the bootstrap config`?

Reply:
will update this to `calls transaction.authenticate with the same stateToken that the widget was bootstrapped with, in the config object`.